src/language-html/printer-html.js

```diff
@@ -1082,9 +1083,16 @@ function printEmbeddedAttributeValue(node, originalTextToDoc, options) {
     if (node.fullName === "v-for") {
       return printVueFor(getValue(), textToDoc);
     }
-
-    if (node.fullName === "slot-scope") {
-      return printVueSlotScope(getValue(), textToDoc);
+    if (
+      node.fullName.charAt(0) === "#" ||
+      node.fullName === "slot-scope" ||
+      node.fullName === "v-slot" ||
+      node.fullName.startsWith("v-slot:") ||
```

Comment:
What do you think about to extract as function named `isSlotNode` and `isSfcSyntaxSugarNode`  to improve readability? like below:

```js
function isSlotNode(node) {
  return (
    node.fullName.charAt(0) === "#" ||
    node.fullName === "slot-scope" ||
    node.fullName === "v-slot" ||
    node.fullName.startsWith("v-slot:")
  );
}
```
```js
// https://github.com/vuejs/rfcs/blob/sfc-improvements/active-rfcs/0000-sfc-script-setup.md
// https://github.com/vuejs/rfcs/blob/sfc-improvements/active-rfcs/0000-sfc-style-variables.md
function isSfcSyntaxSugarNode(node, options) {
  return (
    isVueSfcBlock(node.parent, options) &&
    ((node.parent.fullName === "script" && node.fullName === "setup") ||
      (node.parent.fullName === "style" && node.fullName === "vars"))
  );
}
```

Reply:
Done